### ISSUE TITLE
fix for the screen readers on windows

### ISSUE THREAD

**Issue by aryanchoudharypro** [axcore/tartube#647]
hi, i am a blind windows user. i want to use this app, but my screen reader, nvda doesn't work with this app. i am on the latest stable release and the windows version is 11

**Comment by aryanchoudharypro** [comment]
understood, thanks. i have tryed the downloaders but nun of them work. but i atleast know where to give feedback!

**Comment by axcore** [comment]
There is a comprehensive list of GUIs [here](https://www.reddit.com/r/youtubedl/wiki/info-guis/) (but it's a long read).

**Comment by aryanchoudharypro** [comment]
about this list? i came to tartube through list and non of the items works well with the screen readers

**Comment by axcore** [comment]
It's worth asking the question on [that subreddit](https://www.reddit.com/r/youtubedl/?rdt=35440), there are bound to be some people with expertise in screen readers.
